math/geometry/trigan/main_function.go

```diff
@@ -0,0 +1,17 @@
+// Package trigan for trigonometric functions and algorithms
+package trigan
+
+import "math"
+
+// Cos  returns the cosine of the radian argument x. [See more](https://en.wikipedia.org/wiki/Sine_and_cosine)
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see sqrt_test.go
+
+func Cos(x float64) float64 {
```

Comment:
Add comments explaining your code.

Reply:
Added a link to the idea that I took as a basis